src/lib/homestore/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ target_sources("${PROJECT_NAME}_homestore" PRIVATE
 		homeobject.cpp
 		blob_manager.cpp
 		shard_manager.cpp
+    heap_chunk_selector.cpp
```

Comment:
formatting

Reply:
Does it make sense to apply some unified format plugin and apply to all? So that when another one is making changes to same file, it won't yield code changes for all the lines.

Reply:
sorry for this. it is formatted in my vscode before pushing the PR, i am not sure why it is not formatted here. will check it again

Reply:
It seems this file use TAB instead of space.  I think we should expand TAB with some space (ts:4 or ts:2, based on teams preference).  It seems like @JacksonYao287 's IDE configured with ts=2

Reply:
i tried several times, it is looks good in my local IDE, not sure why it is like this in github show page

Reply:
try `retab` after setting `et` and `ts=4`.